intranet/scss/components/_block-list-teaser.scss

```diff
@@ -8,22 +8,51 @@
     float: left;
   }
 
+  &__comment-count {
+    float: right;
+
+    a {
```

Comment:
change to a {
      &,
      &:focus,
      &:visited,
      &:hover {
        color: $green-64;
    }
   }